### ISSUE TITLE
Accessibility - Screen readers/Colour blindness

### ISSUE THREAD

**Issue by DanielTung** [University-of-Portsmouth/myport#1]
- Icons, when run through a screen reader such as JAWS, have no description. Aria tags are missing.
- When run through different colour filters, to simulate different visual impairments, elements such as the footer or navigation hover over become hard to differentiate.

**Comment by zoetrope69** [comment]
Cheers for noting these, will make sure to correct these.

**Comment by johnwheal** [comment]
Sorry for the delay in replying - I dealt with this a couple of weeks ago but forgot to update this issue.

I've added Aria tags and tested with a screen reader. I don't have access to JAWS so it may be worth double checking.

I've tested using various colour filters and made improvements.

**Comment by DanielTung** [comment]
Thanks, seems to be a lot better. There are a few places tags could be added but for the most part the page is now accessible for screen reader users.
